_data-prepper/pipelines/expression-syntax.md

```diff
@@ -134,79 +86,82 @@ not /status_code in {200, 202}
 /response != null
 ```
 
-## Definitions
+### Reserved symbols
 
-This section provides expression definitions. 
+Reserved symbols refer to a set of symbols that are not currently used in the expression syntax but are reserved for possible future functionality or extensions. The reserved symbols include `^`, `*`, `/`, `%`, `+`, `-`, `xor`, `=`, `+=`, `-=`, `*=`, `/=`, `%=`, `++`, `--`, and `${<text>}`.
 
-### Literal
-A literal is a fundamental value that has no children:
-- Float: Supports values from 3.40282347 &times; 10<sup>38</sup> to 1.40239846 &times; 10<sup>&minus;45</sup>.
-- Integer: Supports values from &minus;2,147,483,648 to 2,147,483,647.
-- Boolean: Supports true or false.
-- JSON Pointer: See the [JSON Pointer](#json-pointer) section for details.
-- String: Supports valid Java strings.
-- Null: Supports null check to see whether a JSON Pointer exists.
+## Syntax components
 
-### Expression string
-An expression string takes the highest priority in a Data Prepper expression and only supports one expression string resulting in a return value. An _expression string_ is not the same as an _expression_.
+Syntax components are the building blocks of expressions in Data Prepper. They allow you to define sets, specify evaluation order, reference values within events, use literal values, and follow specific white space rules. Understanding these components is crucial for creating and working with expressions effectively in Data Prepper pipelines.
 
-### Statement
-A statement is the highest-priority component of an expression string.
+### Set initializers
```

Comment:
These (set initializers) don't actually work. We should remove them from the documentation.